dockerfile_build/test/BUILD

```diff
@@ -13,10 +13,3 @@ py_binary(
     name = "config_stripper",
     srcs = ["//dockerfile_build:config_stripper.py"],
 )
-
-dockerfile_build(
```

Comment:
This rule is used in //reproducible/ubunut:ubuntu_build currently

Reply:
I think this one actually isn't - there's a duplicate one up in dockerfile_build/BUILD that is. I think it was just a copy/paste error when you split tests.

Reply:
Ah sorry, didn't realize this was /test